SQL/1hgj.sql

```diff
@@ -624,6 +624,7 @@ VALUES
 (null,Now(),'-1','MINIMUM_DISPLAY_NAME_LENGTH', '1', 'USERS', 'Shortest user display name', 'NUMBER', '[]', '1', '1', '1'),
 (null,Now(),'-1','MAXIMUM_DISPLAY_NAME_LENGTH', '50', 'USERS', 'Longest user display name', 'NUMBER', '[]', '1', '1', '1'),
 (null,Now(),'-1','MAX_SCREENSHOT_FILE_SIZE_IN_BYTES', '5000000', 'JAM_SETTINGS', 'Maximum screenshot file size in bytes', 'NUMBER', '[]', '0', '1', '1'),
+(null,Now(),'-1','MAX_ASSET_FILE_SIZE_IN_BYTES', '5000000', 'ASSETS', 'Maximum asset file size in bytes', 'NUMBER', '[]', '0', '1', '1'),
```

Comment:
Nice find, but can you please change it to 15MB?

Reply:
I also changed screenshot file size to an even 5MB